### PR TITLE
Test for Chat components

### PR DESCRIPTION
**Pull request by olaore**
# Description

Describe change or new feature here.

Added test for the new chat components.

## Type of change

Please delete options that are not relevant.

- [x] Test

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

## Linkt to Trello card
https://trello.com/c/l4bV4JXK/82-test-frontend-layout-code-for-chat-interface

### CODE REVIEW

**Comment by olaore** [comment]
[Test frontend layout code for chat interface](https://trello.com/c/l4bV4JXK/82-test-frontend-layout-code-for-chat-interface)

**Review by cappers86 — APPROVED** [review]
nice

File: src/components/Layout/Chat/Chat.jsx [hunk 1]
```diff
@@ -62,8 +62,8 @@ const ChatHeader = styled.div`
 
 const ChatBody = styled.div`
   position: relative;
-  height: 100%;
-  overflow-y: auto;
+  height: 250px;
+  overflow-y: scroll;
```
Comment (rodrigograca31) [inline src/components/Layout/Chat/Chat.jsx]: Why was this changed? :thinking: 
That wasnt your job...... :joy: 

Unless you can justify this CSS change I wont accept it. :)

**Review by rodrigograca31 — CHANGES_REQUESTED** [review]

File: src/components/Layout/Chat/Chat.jsx (hunk 1 above)
Comment (olaore) [reply src/components/Layout/Chat/Chat.jsx]: I was playing around with the autoscroll feature on Friday 🤦🏾‍♂️.
Removing now

**Review by rodrigograca31 — APPROVED** [review]
